Helper/DeviceView.php

```diff
@@ -54,14 +54,12 @@ class DeviceView
      */
     public function __construct(Container $serviceContainer)
     {
-        if (false === $serviceContainer->isScopeActive('request')) {
+        if (!$this->request = $serviceContainer->get('request_stack')->getMasterRequest()) {
```

Comment:
Well, Symfony 2.3 is still supported until May and will then receive security fixes for another year. So personally, I would not make that change yet.